### PR TITLE
[MDCT-1869] Section D: Program Integrity

### PR DESCRIPTION
**Pull request by juliuscaesar**
## Description
Adds Section D: Program Integrity. Closes [MDCT-1869](https://qmacbis.atlassian.net/browse/MDCT-1869)

### How to test
http://localhost:3000/mcpar/plan-level-indicators/program-integrity

### Changed Dependencies
N/A

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by scwambach — APPROVED** [review]

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 1]
```diff
@@ -1,18 +1,181 @@
...
+              "children": [
+                {
+                  "id": "dpi-7-o1",
+                  "type": "number",
+                  "props": {
+                    "label": "D1.X.7 Count of program integrity referrals to the state",
+                    "hint": "Enter the count of program integrity referrals that the plan made to the state in the past year. Enter the count of referrals made.",
+                    "mask": "comma-separated"
+                  }
+                }
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:86]: Well these are a super weird case, aren't they? After going back and forth a bunch, I think we should make the IDs for these 3 `dpi-7`s `dpi-7a`, `dpi-7b`, and `dpi-7c` to differentiate them from true options/choices and make it clear they are their own fields.

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 2]
```diff
@@ -1,18 +1,181 @@
...
+          "label": "D1.X.6 Referral path for program integrity referrals to the state",
+          "hint": "What is the referral path that the plan uses to make program integrity referrals to the state? Select one.",
+          "choices": [
+            {
+              "name": "dpi-6-o1",
+              "type": "choice",
+              "label": "Makes referrals to the Medicaid Fraud Control Unit (MFCU) only",
+              "children": [
+                {
+                  "id": "dpi-7-o1",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:79]:
```suggestion
                  "id": "dpi-7a",
```

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 3]
```diff
@@ -1,18 +1,181 @@
...
+                }
+              ]
+            },
+            {
+              "name": "dpi-6-o2",
+              "type": "choice",
+              "label": "Makes referrals to the State Medicaid Agency (SMA) and MFCU concurrently",
+              "children": [
+                {
+                  "id": "dpi-7-o2",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:95]:
```suggestion
                  "id": "dpi-7b",
```

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 4]
```diff
@@ -1,18 +1,181 @@
...
+                }
+              ]
+            },
+            {
+              "name": "dpi-6-o3",
+              "type": "choice",
+              "label": "Makes some referrals to the SMA and others directly to the MFCU",
+              "children": [
+                {
+                  "id": "dpi-7-o3",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:111]:
```suggestion
                  "id": "dpi-7c",
```

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 5]
```diff
@@ -1,18 +1,181 @@
...
+          "hint": "What is the ratio of program integrity referrals listed in the previous indicator made to the state in the past year per 1,000 beneficiaries, using the plan's total enrollment as of the first day of the last month of the reporting year (reported in indicator D1.I.2) as the denominator.",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-9",
+        "type": "textarea",
+        "props": {
+          "label": "D1.X.9 Plan overpayment reporting to the state",
+          "hint": "Describe the plan’s latest annual overpayment recovery report submitted to the state as required under 42 CFR 438.608(d)(3).</p>Include, for example, the following information:<ul><li>The date of the report (rating period or calendar year).</li><li>The dollar amount of overpayments recovered.</li><li>The ratio of the dollar amount of overpayments recovered as a percent of premium revenue as defined in MLR reporting under 438.8(f)(2).</li></ul>"
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:138]: Right now with no opening `p` tag it renders like this:

Screen Shot 2022-08-26 at 08 45 28

Suggest either adding an opening `<p>` or changing the closing `</p>` to a line break.

File: services/ui-src/src/forms/mcpar/dpi/dpi.json [hunk 6]
```diff
@@ -1,18 +1,181 @@
...
+        }
+      },
+      {
+        "id": "dpi-10",
+        "type": "radio",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "",
+          "hint": "",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.json:146]: Oops. Just forgot to fill these in.

```suggestion
          "label": "D1.X.10 Changes in beneficiary circumstances",
          "hint": "Select the frequency the plan reports changes in beneficiary circumstances to the state.",
```

File: services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts [hunk 7]
```diff
@@ -1,3 +1,29 @@
...
 
-export default object({});
+export default object({
+  "dpi-1": number(),
+  "dpi-2": number(),
+  "dpi-3": number(),
+  "dpi-4": number(),
+  "dpi-5": number(),
+  "dpi-6": radio(),
+  "dpi-7-o1": nested(
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts:11]:
```suggestion
  "dpi-7a": nested(
```

**Review by gmrabian — CHANGES_REQUESTED** [review]
Missing the question 10 label and hint text
Screen Shot 2022-08-26 at 7 48 47 AM
Screen Shot 2022-08-26 at 7 49 10 AM

File: services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts [hunk 8]
```diff
@@ -1,3 +1,29 @@
...
+    number,
+    "dpi-6",
+    "Makes referrals to the Medicaid Fraud Control Unit (MFCU) only"
+  ),
+  "dpi-7-o2": nested(
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts:16]:
```suggestion
  "dpi-7b": nested(
```

File: services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts [hunk 9]
```diff
@@ -1,3 +1,29 @@
...
+    number,
+    "dpi-6",
+    "Makes referrals to the State Medicaid Agency (SMA) and MFCU concurrently"
+  ),
+  "dpi-7-o3": nested(
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpi/dpi.schema.ts:21]:
```suggestion
  "dpi-7c": nested(
```

**Review by braxex — CHANGES_REQUESTED** [review]
Just a couple changes needed.

**Review by braxex — APPROVED** [review]

**Pull request merged**